src/main/scala/it/pagopa/interop/backendforfrontend/service/CatalogProcessService.scala

```diff
@@ -7,28 +7,32 @@ import scala.concurrent.Future
 
 trait CatalogProcessService {
 
-  def createEService(eServiceSeed: EServiceSeed)(implicit contexts: Seq[(String, String)]): Future[EService]
-
   def activateDescriptor(eServiceId: String, descriptorId: String)(implicit
     contexts: Seq[(String, String)]
   ): Future[Unit]
 
-  def publishDescriptor(eServiceId: String, descriptorId: String)(implicit
-    contexts: Seq[(String, String)]
-  ): Future[Unit]
+  def createDescriptor(eServiceId: UUID, eServiceDescriptorSeed: EServiceDescriptorSeed)(implicit
+                                                                                         contexts: Seq[(String, String)]
+  ): Future[EServiceDescriptor]
 
-  def getEServices(
-    name: Option[String] = None,
-    eServicesIds: Seq[UUID],
-    producersIds: Seq[UUID],
-    states: Seq[EServiceDescriptorState],
-    offset: Int,
-    limit: Int
-  )(implicit contexts: Seq[(String, String)]): Future[EServices]
+  def createEService(eServiceSeed: EServiceSeed)(implicit contexts: Seq[(String, String)]): Future[EService]
```

Comment:
please do not move code 🙏 
can you place them back?

PS: this file needs formatting